tests/unit/amo/components/TestCategories.js

```diff
@@ -304,6 +304,47 @@ describe(__filename, () => {
     expect(root.find(ErrorList)).toHaveLength(1);
   });
 
+  it('renders a class name with a color for each category', () => {
+    const categoriesResponse = {
+      // Generate 13 categories. We use 13 ordered letters because the reducer
+      // sorts the categories by name (alphabetically). By doing this here, we
+      // ensure a consistent output, which we need to make sure the 1st and
+      // 13rd categories have the correct CSS class names...
```

Comment:
typo: 13th like Friday The 13th 👻 

![jason-I-love-it](https://user-images.githubusercontent.com/55398/67426460-bfeb8800-f59f-11e9-868f-bdd3901c4a53.jpg)

Reply:
Yikes, sorry.